CAN-231: Account Settings: Implement API to show list of Direct suppo…

vikasranatalentelgia opened the pull request:
…rted camps

Account Settings: Implement API to show list of Direct supported camps

vikasranatalentelgia committed:
CAN-231: Account Settings: Implement API to show list of Direct supported camps

Account Settings: Implement API to show list of Direct supported camps

akuks commented on src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx:
```diff
@@ -161,7 +161,7 @@ export default function DelegatedSupportCampsUI({
               Are you sure, you want to remove your delegate support given to
               pranav_telentelgia under the topic{" "}
```
`pranav_talentelgia` is it a typo? or we need it

vikasranatalentelgia replied:
This is static content. Once APIs are ready it will come from the apis

akuks commented on src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx:
```diff
@@ -7,120 +7,78 @@ export default function DirectSupportedCampsUI({
+                <CardTitle title_link={data.title_link} value={data.title} />
+              }
+              extra={
+                <a
```
Can we use Link here?

vikasranatalentelgia replied:
Changed to div

akuks commented on src/components/ComponentPages/DirectSupportedCamps/index.tsx:
```diff
@@ -13,11 +14,25 @@ const DirectSupportedCamps = () => {
+
+  const fetchDirectSupportedCampsList = async () => {
+    let response = await getDirectSupportedCampsList();
+    if (response && response.status_code === 200) {
```
I think the `response.status_code === 200` will work and we can remove the `response` inside `if` condition. No?

vikasranatalentelgia replied:
response could be null or empty or undefined in case of connection error or any other reason. For that reason I have applied it inside the if condition

akuks reviewed (changes requested):
Please go through the comments.

vikasranatalentelgia committed:
Code refactor

Code refactor

vikasranatalentelgia committed:
Merge branch 'development' into CAN-204

vikasranatalentelgia committed:
CAN-227, CAN-228 : Test Cases for Supported Camps

Account Setting: Frontend Jest test cases for delegated support camps
Account Setting: Frontend Jest test cases for direct support camps

vikasranatalentelgia committed:
CAN-232: Implement API to show list of delegated supported camps

Account Settings: Implement API to show list of delegated supported camps

vikasranatalentelgia committed:
Supported Camps: Nickname and Support delegated to as per API changes

Supported Camps: Nickname and Support delegated to as per API changes
